src/pool-cl/CLPositionManager.sol

```diff
@@ -402,7 +402,7 @@ contract CLPositionManager is
             uint128 liquidity,
             uint256 feeGrowthInside0LastX128,
             uint256 feeGrowthInside1LastX128,
-            bool hasSubscriber
+            address _subscriber
```

Comment:
wonder if we can just return `ICLSubscriber`

Reply:
agree, updated